### ISSUE TITLE
Reusing email field causes SQL error

### ISSUE THREAD

**Issue by joshuabaker** [verbb/formie#581]
###  Description

Reusing an email field causes the follow SQL error.

```
PDOException: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'field_emailAddress' in 'field list' in /srv/app/myapp/htdocs/api/vendor/yiisoft/yii2/db/Command.php:1302
```

Looking at the stack trace it seems like there’s a check happening to ensure the email is unique, which is how it’s configured in the other form. Reviewing the settings in the new form the ‘Unique Value’ is unchecked.

### Steps to reproduce
1. Create an email field in one form
2. Select that field in a second form

### Form settings

- Multi-page form: Yes
- Submission Method: Ajax
- Client-side Validation: Yes

### Additional info
- Plugin version: 1.4.13
- Craft version: 3.7.9
- Multi-site: Yes

### Stack trace

```
#0 /srv/app/myapp/htdocs/api/vendor/yiisoft/yii2/db/Command.php(1302): PDOStatement->execute()
#1 /srv/app/myapp/htdocs/api/vendor/yiisoft/yii2/db/Command.php(1168): yii\db\Command->internalExecute('SELECT EXISTS(S...')
#2 /srv/app/myapp/htdocs/api/vendor/yiisoft/yii2/db/Command.php(436): yii\db\Command->queryInternal('fetchColumn', 0)
#3 /srv/app/myapp/htdocs/api/vendor/yiisoft/yii2/db/Query.php(446): yii\db\Command->queryScalar()
#4 /srv/app/myapp/htdocs/api/vendor/craftcms/cms/src/db/Query.php(213): yii\db\Query->exists(NULL)
#5 /srv/app/myapp/htdocs/api/vendor/verbb/formie/src/fields/formfields/Email.php(116): craft\db\Query->exists()
#6 /srv/app/myapp/htdocs/api/vendor/craftcms/cms/src/base/Element.php(1992): verbb\formie\fields\formfields\Email->validateUniqueEmail(Object(verbb\formie\elements\Submission), NULL)
#7 /srv/app/myapp/htdocs/api/vendor/yiisoft/yii2/validators/InlineValidator.php(84): craft\base\Element->validateCustomFieldAttribute('field:emailAddr...', Array, Object(yii\validators\InlineValidator), 'jane.doe@exampl...')
#8 /srv/app/myapp/htdocs/api/vendor/yiisoft/yii2/validators/Validator.php(261): yii\validators\InlineValidator->validateAttribute(Object(verbb\formie\elements\Submission), 'field:emailAddr...')
#9 /srv/app/myapp/htdocs/api/vendor/yiisoft/yii2/base/Model.php(368): yii\validators\Validator->validateAttributes(Object(verbb\formie\elements\Submission), Array)
#10 /srv/app/myapp/htdocs/api/vendor/verbb/formie/src/elements/Submission.php(398): yii\base\Model->validate(Array, true)
#11 /srv/app/myapp/htdocs/api/vendor/verbb/formie/src/controllers/SubmissionsController.php(461): verbb\formie\elements\Submission->validate()
#12 [internal function]: verbb\formie\controllers\SubmissionsController->actionSubmit()
#13 /srv/app/myapp/htdocs/api/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#14 /srv/app/myapp/htdocs/api/vendor/yiisoft/yii2/base/Controller.php(181): yii\base\InlineAction->runWithParams(Array)
#15 /srv/app/myapp/htdocs/api/vendor/craftcms/cms/src/web/Controller.php(190): yii\base\Controller->runAction('submit', Array)
#16 /srv/app/myapp/htdocs/api/vendor/yiisoft/yii2/base/Module.php(534): craft\web\Controller->runAction('submit', Array)
#17 /srv/app/myapp/htdocs/api/vendor/craftcms/cms/src/web/Application.php(276): yii\base\Module->runAction('formie/submissi...', Array)
#18 /srv/app/myapp/htdocs/api/vendor/yiisoft/yii2/web/Application.php(104): craft\web\Application->runAction('formie/submissi...', Array)
#19 /srv/app/myapp/htdocs/api/vendor/craftcms/cms/src/web/Application.php(261): yii\web\Application->handleRequest(Object(craft\web\Request))
#20 /srv/app/myapp/htdocs/api/vendor/yiisoft/yii2/base/Application.php(392): craft\web\Application->handleRequest(Object(craft\web\Request))
#21 /srv/app/myapp/htdocs/api/web/index.php(22): yii\base\Application->run()
#22 {main}
```

**Comment by engram-design** [comment]
So you receive this error when saving the form, or when making a submission on the form?

Does the source email field have "Unique Value" checked?

Lastly, are you creating a new field from the existing one, or using a synced field?

**Comment by joshuabaker** [comment]
**So you receive this error when saving the form, or when making a submission on the form?**
Submitting the form/creating a submission.

**Does the source email field have "Unique Value" checked?**
The original email field had ‘Unique Value’ checked. When reused via ‘Add existing fields…’ it did not have that checked.

**Lastly, are you creating a new field from the existing one, or using a synced field?**
In my testing, I added it as a new field.

**Comment by joshuabaker** [comment]
N.B. This has cropped up on another site. It _may_ be unrelated to the ‘Add existing fields...’ functionality as the client is convinced that they added a fresh field (also labelled “Email Address”).

**Comment by engram-design** [comment]
Ah, yep this is an issue with Craft 3.7+ due to their field handle changes. Should be an issue for **any** new email field created with "Unique Value" checked. Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.17",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.18](https://github.com/verbb/formie/releases/tag/1.4.18)
